Library/Homebrew/cmd/version.rb

```diff
@@ -0,0 +1,11 @@
+module Homebrew
+  def version
+    if ARGV.named.empty?
```

Comment:
Yes, but I thought it would be better to do something useful instead of failing. Would you prefer it to fail instead?